cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -136,7 +136,8 @@ def _set_schema_def(self):
                     f"Validation cannot be performed."
                 )
             else:
-                self.schema_def = schema.get_schema_definition(version)
+                if not self.schema_def:
```

Comment:
Required so that `_set_schema_def` can be called externally (by tests) to enable mocking. Without this line, it would be overwritten each time `validate_adata()` is called